### DISCUSSION TITLE
sqlite3报错

### DISCUSSION THREAD

**Discussion by thbun** [jxxghp/MoviePilot#2201] [问题咨询]
<img width="1038" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/50077379/85c9e159-9450-493b-af79-e7239b6988cd">=
这个我该如何解决呢？

**Comment by iamckun** [comment]
卡io了吧，我也经常这样，想让支持更多的数据库直接吧数据库单独部署就不会让pt和数据库抢磁盘的io了，但是作者没回应加自定义数据库的支持
